src/Tests/FileEntityUnitTest.php

```diff
@@ -59,24 +59,28 @@ function testImageDimensions() {
     // Test hook_file_insert().
     // Files have been saved as part of setup (in FileEntityTestHelper::setUpFiles).
     foreach ($this->files['image'] as $file) {
-      $files[$file->id()] = $file->metadata;
+      $files[$file->id()] = $file->getAllMetadata();
+      $height = $file->getMetadata('height');
+      $width = $file->getMetadata('width');
       $this->assertTrue(
-        isset($file->metadata['height']),
+        isset($height),
```

Comment:
If you have a hasMetadata() method then you can just use that instead of the variable and isset().